<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CActionEditDialog</class>
 <widget class="QDialog" name="CActionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>执行内容</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_delayBefore">
     <property name="title">
      <string>执行前延时</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTimeEdit" name="timeEdit_delayBefore">
        <property name="minimumTime">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>1</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>230</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_delayBefore">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton_exitAction">
        <property name="text">
         <string>退出AutoDoIt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_poweroffAction">
        <property name="text">
         <string>关机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_powerOff">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="radioButton_powerOff">
             <property name="text">
              <string>关机</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_hibernate">
             <property name="text">
              <string>休眠</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_suspend">
             <property name="text">
              <string>睡眠</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_startStopAction">
        <property name="text">
         <string>启动/停止任务</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_startStop">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton_startTask">
             <property name="text">
              <string>启动任务</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_stopTask">
             <property name="text">
              <string>停止任务</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_messageAction">
        <property name="text">
         <string>显示消息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_message">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioButton_message">
             <property name="text">
              <string>弹窗消息</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_notify">
             <property name="text">
              <string>托盘消息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_message">
           <property name="placeholderText">
            <string>消息</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_startStopProcessAction">
        <property name="text">
         <string>执行/停止外部程序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_startStopProcess">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButton_startProcess">
             <property name="text">
              <string>执行外部程序</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_stopProcess">
             <property name="text">
              <string>停止外部程序</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="lineEdit_startStopProcess">
             <property name="placeholderText">
              <string>外部程序及参数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_selectProcess">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../qrc.qrc">
               <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_delayAfter">
     <property name="title">
      <string>执行后延时</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTimeEdit" name="timeEdit_delayAfter">
        <property name="minimumTime">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>1</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>230</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_delayAfter">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_error">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CActionEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_messageAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_message</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_startStopAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_startStop</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_startStopProcessAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_startStopProcess</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_poweroffAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_powerOff</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
